mailpoet/tests/integration/Subscription/ManageTest.php

```diff
@@ -64,59 +79,65 @@ public function testItDoesntRemoveHiddenSegmentsAndCanResubscribe() {
       $this->settings,
       $this->diContainer->get(NewSubscriberNotificationMailer::class),
       $this->diContainer->get(WelcomeScheduler::class),
+      $this->diContainer->get(CustomFieldsRepository::class),
```

Comment:
I think that this could all be replaced with a helper I added to the codebase a little while ago and I'm just curious to know your opinion on it. With the helper I think it would look like this:

```php
$manage = $this->getServiceWithOverrides(Manage::class, [
  'urlHelper' => Stub::make(UrlHelper::class, [
    'redirectBack' => null,
  ]),
  'fieldNameObfuscator' => Stub::make(FieldNameObfuscator::class, [
    'deobfuscateFormPayload' => function($data) {
      return $data;
    },
  ]),
  'linkTokens' => Stub::make(LinkTokens::class, [
    'verifyToken' => function($token) {
      return true;
    },
  ]),
  'settings' => $this->settings
]);
```

Do you have a preference for one approach vs the other?

Definitely not a required change!

Reply:
We can simplify the code when it's possible. This pull request isn't big, so I would replace it with your new method.